analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -76,6 +78,13 @@ class Sbt(
 
     override fun command(workingDir: File?) = if (Os.isWindows) "sbt.bat" else "sbt"
 
+    override fun getVersion(workingDir: File?): String {
+        // Avoid newer Sbt versions to warn about "Neither build.sbt nor a 'project' directory in the current directory"
+        // and prompt the user to continue or quit on Windows where the "-batch" option is not supported.
+        val dummyProjectDir = createTempDir(ORT_NAME, managerName).also { it.resolve("build.sbt").createNewFile() }
+        return super.getVersion(dummyProjectDir).also { dummyProjectDir.safeDeleteRecursively() }
```

Comment:
I think this workaround should only be applied if `workingDir` is null, because if this is called from `checkVersion` as part of `beforeResolution` it should check the SBT version in the directory of the definition file.

I will already rebase my azure branch on top of this to verify that it fixes the issue there.

Reply:
> I think this workaround should only be applied if `workingDir` is null

Good point, done.

Reply:
I can confirm this fixed the issue in the Azure build I'm working on.